apps/prairielearn/src/ee/pages/instructorAiGenerateJobs/instructorAiGenerateJobs.html.ts

```diff
@@ -0,0 +1,74 @@
+import { formatDate } from '@prairielearn/formatter';
+import { html } from '@prairielearn/html';
+import { renderEjs } from '@prairielearn/html-ejs';
+
+import { HeadContents } from '../../../components/HeadContents.html.js';
+import { JobStatus } from '../../../components/JobStatus.html.js';
+import { Job, User } from '../../../lib/db-types.js';
+
+export function InstructorAIGenerateJobs({
+  resLocals,
+  jobs,
+}: {
+  resLocals: Record<string, any>;
+  jobs: { job: Job; user: User | null }[];
```

Comment:
Rather than redeclaring this type, move the `JobRowSchema` object here and use `z.infer` to derive a type from it. This ensures that the types easily stay in sync. See other pages for examples of this.

Reply:
changed in a133e9b738f1eb22f09f6fa83561847be451b8eb